src/redux/Missions/missions.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable camelcase */
+// import axios from 'axios';
+// CONSTANTS
+const INITIAL_STATE = {
+  missions: [],
+};
+const FETCH_MISSIONS = 'Missions/FETCH_MISSIONS';
+const TOGGLE_MISSION = 'Missions/TOGGLE_MISSION';
+
+// Local ACTIONS
+
+// API ACTIONS
+// export const fetchMissions = () => (dispatch) => {
+//   dispatch({ type: GET_MISSIONS_START });
+//   axios.get(`${process.env.REACT_APP_SPACEX_BASE_API}missions`).then((response) => {
+//     dispatch({ type: GET_MISSIONS_SUCCESS, payload: response.data });
+//   }).catch((error) => { dispatch({ type: GET_MISSIONS_ERROR, payload: error }); });
+// };
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable.

Reply:
Ooh, we forgot to remove those when cleaning things up. 👍